app/controllers/orders_controller.rb

```diff
@@ -24,6 +24,11 @@ def create
   end
 
   def destroy
+    @current_order.destroy
+    flash[:status] = :success
```

Comment:
I think based on the flash HTML that Mina set up, this would need to be flash[:success] = "Successfully destroyed..." (in order to have the flash message show up green)